client/web/compose/src/components/PageBlocks/ChartConfigurator.vue

```diff
@@ -52,6 +53,14 @@
           :reduce="option => option.blockID"
           :clearable="true"
           :placeholder="$t('chart.drillDown.openInModal')"
+          class="mb-2"
+        />
+
+        <column-picker
```

Comment:
Column picker should be an input group as the chart editor above, same with Metric
It should also be disabled if a block is selected (since its useless then)